h/search/__init__.py

```diff
@@ -16,6 +17,7 @@
 def includeme(config):
     settings = config.registry.settings
     settings.setdefault('es.host', 'http://localhost:9200')
+    settings.setdefault('es.url', 'http://localhost:9201')
```

Comment:
I think we could swap these soon if we want now that we actually have a real connection working. I'm not sure what that would look like/how difficult that would be in production?